index.html

```diff
@@ -0,0 +1,308 @@
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="see-list">
+                <a href="" class="view-all">SEE THE WHOLE PROGRAM</a>
+            </div>
+            <div class="join-us">
+                <button class="action">Join Our Upcoming Cohort</button>
+            </div>
+        </div>
+    </section>
+
+    <!-- featured speakers section -->
+    <section class="tutors">
+        <br>
+        <br>
+        <div class="pat-title speaker">
+            <h2 class="sp-tag">Featured Tutors</h2>
+            <div class="orange-line">
+                <hr class="line">
+            </div>
+        </div>
+
+        <div class="speaker-container">
+            <!-- <div class="speaker-info">
+                <div class="speaker-img">
+                    <img class="thumbnail1_1" src="images/tutor2.png" alt="image of speaker">
+                </div>
+                <div class="bio-container">
+                    <div>
+                        <h4 class="speaker-tag">Lyla Treticov</h4>
+                        <p class="speaker-title">
+                            Secretary General of the Wikimedia Foundation
+                        </p>
+                    </div>
+                    <br>
+                    <div class="speaker-bio">
+                        <p class="name">Layla Treticov is the Executive Director of the Wikimedia Foundation, the
+                            non-profit
+                            organization
+                            that runs Wikipedia. Wikipedia is provided free of charge in 290 languages ​​each month to
+                            over
+                            100 million people, nearly half of the world's population.</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="speaker-info">
+                <div class="speaker-img">
+                    <img class="thumbnail1_1" src="images/tutor3.png" alt="image of speaker">
+                </div>
+                <div class="bio-container">
+                    <div>
+                        <h4 class="speaker-tag">Lyla Treticov</h4>
+                        <p class="speaker-title">
+                            Secretary General of the Wikimedia Foundation
+                        </p>
+                    </div>
+                    <br>
+                    <div class="speaker-bio">
+                        <p class="name">Layla Treticov is the Executive Director of the Wikimedia Foundation, the
+                            non-profit
+                            organization
+                            that runs Wikipedia. Wikipedia is provided free of charge in 290 languages ​​each month to
+                            over
+                            100 million people, nearly half of the world's population.</p>
+                    </div>
+                </div>
+            </div> -->
```

Comment:
- [ ] As requested by the previous code reviewer, Please make sure you remove commented code from your project to make your code clean. :pray:  You should not push commented code to your repository :smile: . You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)